.github/workflows/build.yml

```diff
@@ -12,10 +12,10 @@ on:
       - develop
 
 jobs:
-  Windows_2019_32bit:
-    runs-on: windows-2019
+  Windows_2022_32bit:
```

Comment:
We might need to tell people to use the VC2022 redist, if this is being used on the build - just an update for the README

Reply:
updated README.md and included a blurb about linux